### ISSUE TITLE
Create a review mechanism for new dandisets/assets

### ISSUE THREAD

**Issue by waxlamp** [dandi/dandiarchive#934]
This would be for preventing illegal content from living in the archive, and for banning users who upload such content.

We may need language in the eventual terms of service about this.

**Comment by waxlamp** [comment]
@yarikoptic ^^^

**Comment by yarikoptic** [comment]
Yes, terms of service should clearly state that any inappropriate content (including proprietary one without free license to redistribute) can't be uploaded to the archive. Violators can be banned.
I think it should be some semi automated system - admins can't review all assets. We should come up with some checks (eg .nwb and .h5 files should actually be hdf5 files) to filter out unlikely to be malicious content. Should have some way to whitelist she uploaders or dandisets.
